frontend/src/AlertConfiguration/AlertConfigurationComponents/AlertConfiguration.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, Fragment } from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { makeStyles } from '@material-ui/core/styles';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import ControlPanel from './ControlPanel';
+import Footer from './Footer';
+import ConfigList from './ConfigList';
+
+const useStyles = makeStyles((theme) => ({
+  main: {
+    display: 'flex',
+    flexDirection: 'column',
+    minHeight: '100vh',
+  },
+}));
+
+/** Alert Configuration Feature
+ * Consists of ControlPanel (a control panel used for creating new configurations and deleting old configurations)
+ * as well as a ConfigList (a list of the user's current configurations)
+*/
```

Comment:
nits for comment style: don't put comments on the first line of the comment, make sure the * is in line, and comments should be full sentences (end with a period). e.g. below
```
/** 
 * Alert configuration feature consists of .. configurations).
 */
```